ParseLiveQuery/src/test/java/com/parse/TestParseLiveQueryClient.java

```diff
@@ -498,6 +539,30 @@ private static JSONObject createObjectDeleteMessage(int requestId, ParseObject p
         return jsonObject;
     }
 
+    private static class LoggingCallbacks implements ParseLiveQueryClientCallbacks {
```

Comment:
Ah. Yeah the class still needs to be instantiated in order to use Transcript. It's declared `final`, but not `static`.  So it is still a new Transcript instance for every instance of LoggingCallbacks (as it should be, in order to properly test that the callbacks are invoked when they're supposed to be).

Just want to make sure we're on the same page about the class and field declarations. The class being declared `static` _does not_ mean that the transcript is a `static` field. It's still an instance field.